.github/CHANGELOG.md

```diff
@@ -175,6 +175,9 @@
 
 <h3>Improvements</h3>
 
+* Probabilty methods are handled by `QubitDevice` and device method
+  requirements are modified to simplify plugin development.
```

Comment:
Don't forget to add a link to the PR! In addition, I would move this up to the `Breaking changes` section of the Changelog --- this is the place for all changes that break either the user or developer API